app/controllers/OnlineTestController.scala

```diff
@@ -62,10 +68,10 @@ trait OnlineTestController extends BaseController {
 
   import Commands.Implicits._
 
-  def getOnlineTest(userId: String) = Action.async { implicit request =>
-    onlineTestingService.getPhase1TestProfile(userId).map {
+  def getOnlineTest(applicationId: String) = Action.async { implicit request =>
```

Comment:
Yep, if it's ok we'll log this for tidy up afterwards - now that this is working and needed relatively urgently i'd rather not touch it if i don't have to ;-)